src/test.js

```diff
@@ -728,7 +728,7 @@ export function internalStop( test ) {
 }
 
 // Forcefully release all processing holds.
-function internalRecover( test ) {
+export function internalRecover( test ) {
```

Comment:
Doesn't have to be in this PR, but we should probably separate out the processing start/stop logic from the test logic.